README.md

```diff
@@ -109,13 +109,9 @@ file. Copy that file to `.env` and customize it to your needs.
 
 
 ##  To run the queues
-```
-scripts/run_celery_local.sh
-```
+`make run-celery-local`
 
-```
-scripts/run_celery_beat.sh
-```
+`make run-celery-beat-local`
```

Comment:
It's great to mention the make target instead. As these are one more level of abstraction, we could even just have a one target named `make run-celeries-local`, as it doesn't have to be a one to one mapping with the actual scripts but rather intended usage (run celery setup locally).

Reply:
hrm, running both the beat and celery workers with one make command? I didn't think that would work, but admittedly didn't try! I shall see!

Reply:
oh wow, we can just add "--beat" to the worker script. That was easy.